servlet-feature-pack/src/main/resources/configuration/domain/subsystems.xml

```diff
@@ -7,10 +7,11 @@
         <subsystem supplement="web-build">ee.xml</subsystem>
         <subsystem supplement="domain">elytron.xml</subsystem>
         <subsystem>io.xml</subsystem>
-        <subsystem>jmx.xml</subsystem>
+        <subsystem supplement="domain">jmx.xml</subsystem>
         <subsystem supplement="web-build">naming.xml</subsystem>
         <subsystem>request-controller.xml</subsystem>
         <subsystem supplement="web-build">security.xml</subsystem>
+        <subsystem>security-manager.xml</subsystem>
```

Comment:
Is there a reason to add this? Was it meant to be there and just missing before?

Reply:
@jamezp Yes, there is one, we have been using the jmx default subsystem template for a domain configuration. The jmx domain variant does not include the  `<remoting-connector/>` declaration.

Reply:
The above response does not make sense for your question. I'm sorry @jamezp , I was confused somehow, you are asking for ` <subsystem>security-manager.xml</subsystem>` instead of this change `<subsystem supplement="domain">jmx.xml</subsystem>`.

About the new inclusion of `security-manager`subsystem in the servlet distribution, I added it to the legacy templates because the subsystem was being included using Galleon when I start working on this issue. It was previously crosschecked with @stuartwdouglas , there is no real downside to including it, so in order to resolve the mismatch between both, servlet-legacy and servlet-galleon, I added it to legacy.  Another solution could have been removing it from Galleon, but as I said, since there was no downside and Stuart was happy with the inclusion, I just include it in the legacy.

Reply:
No problem @yersan. As long as the Galleon and legacy distributions match I think it's fine.